src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random;
+    private final ColorSupplier colorSupplier;
+
+    public Lottery(Random random, ColorSupplier colorSupplier) {
+        this.random = random;
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        String color = String.valueOf(colorSupplier.getRandomColor());
+        int number = random.nextInt(100);
```

Comment:
'100' is a magic number here. Please refer to common mistakes to find out how to declare your constants